cli/help.go

```diff
@@ -47,6 +47,7 @@ Options:
                                     {{printf "\t * mesos.address=\taddress to bind on [$SWARM_MESOS_ADDRESS]"}}
                                     {{printf "\t * mesos.port=\tport to bind on [$SWARM_MESOS_PORT]"}}
                                     {{printf "\t * mesos.offertimeout=10m\ttimeout for offers [$SWARM_MESOS_OFFER_TIMEOUT]"}}
+                                    {{printf "\t * mesos.role=\trole for offers [$SWARM_MESOS_ROLE]"}}
```

Comment:
any string is valid, it's defined by the mesos master, so yes `drunk` can be valid.